alerts/AggregatedAPIErrors.md

```diff
@@ -0,0 +1,66 @@
+# AggregatedAPIErrors
+
+## Meaning
+
+This alert is triggered by the [cluster-monitoring-operator](https://github.com/openshift/cluster-monitoring-operator). Whenever a call is made on one of the aggregated openshift-api and an error occurs, a counter is incremented. For the case that two or more errors appeared over the last 5 mins, this alert is fired.
```

Comment:
@paulfantom afaik this isn't directly coming from CMO but just the alerts comes as part of CMO, right? the metrics are from the API server?

Reply:
yes, you are right.

Reply:
When we get detailed about the specific triggers (e.g. "two or more" and "last 5 minutes", I start to worry about not being DRY.  What happens if the alerts evolve, and those evolutions are backported to some z stream.  If we don't cut runbook releases per 4.y.z, we might link folks to incorrect data.  If we do cut releases per 4.y.z, we either have a whole lot of maintenance work on our hands, or we will serve older folks stale docs (where more recent doc changes actually _did_ apply to their 4.y.z, but just haven't been backported yet).  I'd much rather keep the scoping here to the generic "here's roughly what this alert is about", without getting down into the specifics of the trigger (which the alert message can talk about if it turns out to be important).

Reply:
I agree fully. I just wanted to give some numbers as they help to grasp the scope of the impact.
But since they are included in the alert message it is not required here and I will keep it more generic.